react/react1/week1/todo-list/src/index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from "./componate/App";
```

Comment:
misspelling here and in the folder naming
```suggestion
import App from "./components/App";
```